action.yaml

```diff
@@ -0,0 +1,83 @@
+name: 'Generate release notes'
+description: 'Action used to generate release notes and create PR in the docs repo'
+inputs:
+  token:  
+    description: 'GitHub token used for checkout and creating PR'
+    required: true
+    default: ${{ github.token }}
+  docs-repo:
+    description: 'docs repository (in format owner/repo)'
+    required: true
+  base-branch:
+    description: 'branch to create PR against in docs repo'
+    required: true
+    default: master
+  allow-automerge:
+    description: 'allow automatic merge of PR in docs repo'
+    type: boolean
+    default: false
+  release-notes-path:
+    description: 'path to release notes file'
+    default: releases/release-notes.md
+runs:
+  using: 'composite'
+  steps:
+    - name: validate required inputs
+      run: |
+        [[ "${{ inputs.docs-repo }}" ]] || { echo "docs-repo input is empty" ; exit 1; }
+        [[ "${{ github.event.release }}" ]] || { echo "github.event.release is empty. Run action after release is published" ; exit 1; }
```

Comment:
Toto jsi otestoval? Funguje to?

Reply:
Upravil jsem to, aby to checkovalo nazev eventu. Pokud se actiona spusti jindy nez pri published release, tak bude failovat